### PR TITLE
Update with Let's Encrypt new stuff

### PR DESCRIPTION
**Pull request by mib32**
@daniel-sim Could you please review?

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -171,3 +171,16 @@
...
 # require "whenever/capistrano"
+
+namespace :ssl do
+  desc 'Get certificates'
+  task :setup do
+    on roles(:app) do
+      rails_server_names = [fetch(:nginx_api_server_name), fetch(:nginx_static_server_name)].compact.map {|n| "-d #{n}"}.join(' ')
+      execute "sudo mkdir /etc/nginx/ssl/"
+      execute "sudo openssl dhparam -out /etc/nginx/ssl/dhparam.pem 2048"
+      execute  "TODO sudo certbot certonly --webroot --webroot-path=/srv/YOUR_APP_NAME/current/public #{rails_server_names}"
```
Comment (daniel-sim) [inline config/deploy.rb]: @mib32 what does the TODO mean here?

**Review by daniel-sim — COMMENTED** [review]
@mib32 looks good

File: config/deploy.rb (hunk 1 above)
Comment (mib32) [reply config/deploy.rb]: @daniel-sim It's detached from the `YOUR_APP_NAME` at the right. Probably i could bring `TODO` closer to it?

Comment (daniel-sim) [reply config/deploy.rb]: @mib32 ah! yes, something to make it a bit more obvious would be good

**Comment by mib32** [comment]
@daniel-sim Updated according to feedback.

**Review by daniel-sim — APPROVED** [review]

**Pull request merged**